setup.sh

```diff
@@ -350,7 +350,7 @@ COPYLEFT_LICENSE_INCLUDE = 'GPL* LGPL*'
   # Apply patch on top of it allowing to perform build in external source directory
   echo "Applying patch on poky"
   cd $mingw_dir
-  git apply $top_repo_dir/meta-intel-edison/utils/0001-Enable-SDKTAROPTS.patch
+  #git apply $top_repo_dir/meta-intel-edison/utils/0001-Enable-SDKTAROPTS.patch
```

Comment:
I found why the patch doesn't apply. Common stuff has been moved to an .inc. So what to do @alext-mkrs ? Is the patch needed? I can fixup the patch so it applies or drop it entirely.

Reply:
TBH I'd drop that Windows SDK thing in its entirety - I'm not sure how well it really works, I don't know how many people use it and finally I believe having a VM for the Linux SDK if necessary is way simpler these days (and avoids all those potential problems).

I don't really know why that specific patch was necessary (it changes layer's defaults) in the first place, so we may as well skip updating it for now.

Reply:
I already fixed it - but we can drop it any time. That would also require a little more cleanup in the setup script (menu)